src/types/globalTypes.ts

```diff
@@ -43,4 +44,9 @@ export enum PageOrder {
   '/lesson-2-2',
   '/lesson-3',
   '/lesson-4',
+  '/lesson-5',
```

Comment:
I would comment all of these other unnimplemented lessons out for now since clicking next will go to a page that doesn't exist, but I'll just handle that with this [ticket](https://github.com/uclaacm/parcel-pointers/issues/180) since currently the page is inaccessible without directly modifying the url.